src/ncp/ncp_uart.cpp

```diff
@@ -287,6 +287,8 @@ void otPlatLog(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat
     }
     va_end(args);
 
+    otNcpStreamWrite(0, reinterpret_cast<const uint8_t*>("\r\n"), 2);
```

Comment:
I also don't think this is necessary. Each log message is already broken into Spinel properties, right? Windows has no issue handling the messages as is right now.